.github/workflows/do-release.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
are these changes part of your PR?

Reply:
No, it was not changed. only line 40 was changed by you